website/docs/reference/api/legacy/unleash/admin/features.md

```diff
@@ -106,8 +106,8 @@ Response format is the same as `api/admin/features`
 
 ## Fetch specific feature toggle {#fetch-specific-feature-toggle}
 
-:::caution Deprecation notice
-This endpoint is deprecated. Please use the [project-based endpoint to fetch specific toggles](/reference/api/legacy/unleash/admin/features-v2.md#get-toggle) instead.
+:::caution Removal notice
+This endpoint was removed in Unleash v5. Please use the [project-based endpoint to fetch specific toggles](/reference/api/legacy/unleash/admin/features-v2.md#get-toggle) instead.
```

Comment:
Maybe add a note about when it was deprecated too? It's mostly to not make it look like we just remove things willy-nilly.

Also, I suggest leaving a blank line before and after lines starting with `:::` to stop prettier from putting it onto one line when formatting.

```suggestion
:::caution Removal notice

This endpoint was removed in Unleash v5 (deprecated since v4). Please use the [project-based endpoint to fetch specific toggles](/reference/api/legacy/unleash/admin/features-v2.md#get-toggle) instead.

```